common/source/docs/common-rpm.rst

```diff
@@ -114,16 +115,21 @@ Electrical commutation RPM sensors can be added retrospectively using something
 series, that have an auxiliary output, can be configured to output a pulse per commutation.
 
 For clarification, this is not the same as the RPM that can be passed 
-via serial telemetry with ESCs.  For information on how to set up RPM reporting with capable  ESCs, see the :ref:`ESC Telemetry<esc-telemetry>`.
+via serial telemetry with ESCs.  For information on how to set up RPM reporting with capable ESCs, see the :ref:`ESC Telemetry<esc-telemetry>`.
 For information on how to set up RPM logging with BL Heli see the :ref:`BLHeli Telemetry<common-dshot-blheli32-telemetry>`.
 
 The setup for electrical commutation RPM sensors is much the same as hall effect sensors, so the steps above are applicable.  The only difference is the scaling value 
-to be entered in the :ref:`RPM1_SCALING<RPM1_SCALING>` parameter.  Now, the scaling value is a function of the number of poles in the motor and should be the reciprocal of the number of 
+to be entered in the :ref:`RP2_SCALING<RPM2_SCALING>` parameter.  Now, the scaling value is a function of the number of poles in the motor and should be the reciprocal of the number of 
 poles.  E.g. A 4 pole motor will need a scaling value of 0.25.
 
 Optical Sensors
 ===============
 
-Again, the setup of optical sensors is much the same as the hall effect sensor.  The only difference is the scaling value.  :ref:`RPM1_SCALING<RPM1_SCALING>` should be set to be the reciprocal 
-of the number of reflective obstacles that pass the sensor in a single rotation. 
+Again, the setup of optical sensors is much the same as the hall effect sensor.  The only difference is the scaling value.  :ref:`RPM2_SCALING<RPM2_SCALING>` should be set to be the reciprocal 
+of the number of reflective obstacles that pass the sensor in a single rotation.
+
+
+Harmonic Notch Center Frequency
+===============================
 
+If ``RPMx_TYPE`` = 4, then the center frequency of the harmonic notch, however it is obtained (See :ref:`common-imu-notch-filtering`), is reported.
```

Comment:
Maybe remove the sub-clause and move it to the end of the sentence?  Took me a couple of goes reading it to understand what was being said. e.g:
```suggestion
If ``RPMx_TYPE`` = 4, then the center frequency of the harmonic notch is reported.  See :ref:`common-imu-notch-filtering` for how this value is obtained.
```